examples/air-quality-sensor-app/air-quality-sensor-common/src/air-quality-sensor-manager.cpp

```diff
@@ -136,78 +137,99 @@ void AirQualitySensorManager::Init()
 
 void AirQualitySensorManager::OnAirQualityChangeHandler(AirQualityEnum newValue)
 {
-    mAirQualityInstance.UpdateAirQuality(static_cast<AirQualityEnum>(newValue));
-    ChipLogDetail(NotSpecified, "Updated AirQuality value: %huu", newValue);
+    Status status = mAirQualityInstance.UpdateAirQuality(static_cast<AirQualityEnum>(newValue));
+    VerifyOrReturn(
+        Status::Success == status,
+        ChipLogError(NotSpecified, "AirQualitySensor App: Failed to set AirQuality attribute %x", (unsigned int) status));
 }
 
 void AirQualitySensorManager::OnCarbonDioxideMeasurementChangeHandler(float newValue)
 {
-    mCarbonDioxideConcentrationMeasurementInstance.SetMeasuredValue(MakeNullable(newValue));
-    ChipLogDetail(NotSpecified, "Updated Carbon Dioxide: %f", newValue);
+    CHIP_ERROR chipError = mCarbonDioxideConcentrationMeasurementInstance.SetMeasuredValue(MakeNullable(newValue));
+    VerifyOrReturn(CHIP_NO_ERROR == chipError,
+                   ChipLogError(NotSpecified, "AirQualitySensor App: Failed to set CarbonDioxide MeasuredValue attribute %s",
+                                chip::ErrorStr(chipError)));
```

Comment:
```suggestion
                   ChipLogError(NotSpecified, "AirQualitySensor App: Failed to set CarbonDioxide MeasuredValue attribute %" CHIP_ERROR_FORMAT,
                                chipError.Format());
```
And similar in other places.